app_feup/android/app/src/debug/AndroidManifest.xml

```diff
@@ -4,4 +4,6 @@
          to allow setting breakpoints, to provide hot reload, etc.
     -->
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.WRITE_CALENDAR"/>
+    <uses-permission android:name="android.permission.READ_CALENDAR"/>
```

Comment:
Do we need the permissions on both manifests? I was under the impression that the debug manifest was merged with the main one whenever the app was ran in debug mode.